src/Mono.Android/Xamarin.Android.Net/AndroidClientHandler.cs

```diff
@@ -272,10 +277,10 @@ string EncodeUrl (Uri url)
 				if(Logger.LogNet)
 					Logger.Log (LogLevel.Info, LOG_APP, " cancelled");
 
-				httpConnection.Disconnect ();
+				await DisconnectAsync (httpConnection);
```

Comment:
Because there is already `ConfigureAwait (false)` called before, this await is not guaranteed to run on callee context hence missing `ConfigureAwait (false)` is a bug.